packages/common/src/core/schemas/hubproject.ts

```diff
@@ -245,5 +251,20 @@ export const HubProjectEditUiSchema: IUiSchema = {
         },
       ],
     },
+    {
+      type: "Section",
+      labelKey: "{{i18nScope}}.relatedItems.label",
```

Comment:
Looks like the AC calls out that the section should have helper text, so you'll want to add that to the section's `options`. Something like: 
```ts
{
  type: "Section",
  options: {
    helperText: {
      labelKey: "{{i18nScope}}.relatedItems.helperText"
    }
  },
  elements: [ ... ]
}
```